### PR TITLE
[MDB ignore] NRI police MOD

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Adds the NRI police MOD as a replacement of the NRI ERT MOD for the NRI police spawn. Same stats armor-wise, with a slight nerf to its module capacity; and a buff to the cell charge depletion. Also, NRI ERT MOD/frontline MOD had its stats buffed to make it more useful for ERT use; as it's no longer used by both the "antagonists" and the ERT.
I have also taken some liberty in order to replace their belts with a normal security ones; affects nothing balance-wise, but makes them, -the officers-, look better.
image
image
image

## How This Contributes To The Skyrat Roleplay Experience
More immersive and lore-accurate gear good; mostly meant to make sure people roleplay as the complete raiders less.

## Proof of Testing

<details>
<summary>Screenshots/Videos
</summary>

image
</details>

## Changelog

:cl: Stalkeros for the initial sprite, OrionTheFox for the design-accurate resprite that is way better than it initially was.
add: NRI Police MOD
balance: NRI Military/now-ERT MOD buffed as it's no longer meant to be balanced for station antagonists.
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
day 100 stalkeros still holds less than zero gbp

**Comment by Stalkeros2** [comment]
GBP is a social construct and doesn't matter here.

File: modular_skyrat/modules/novaya_ert/code/mod_suit.dm [hunk 1]
```diff
@@ -54,15 +54,15 @@
 	)
 
 /datum/armor/mod_theme_frontline
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/mod_suit.dm:56]: what uses this mod now, the pirates?

**Review by Zonespace27 — APPROVED** [review]
Code good

File: modular_skyrat/modules/novaya_ert/code/mod_suit.dm (hunk 1 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/novaya_ert/code/mod_suit.dm:56]: It's now ERT only, hence the buff; police one's used by the midround spawn.

**Review by Gandalf2k15 — APPROVED** [review]
ok

**Pull request merged**